src/goto-symex/symex_target_equation.cpp

```diff
@@ -623,6 +623,14 @@ void symex_target_equationt::convert_assertions(
   {
     if(step.is_assert())
     {
+      prop_conv.conditional_output(
+        prop_conv.debug(),
+        [&step](messaget::mstreamt &mstream) {
+          std::ostringstream oss;
+          step.output(oss);
```

Comment:
Why not just `step.output(mstream)`?

Reply:
Indeed! I have added a second commit cleaning this up across all similar bits of code in this file.